site-manage/datemanager/tool/src/main/java/org/sakaiproject/datemanager/tool/MainController.java

```diff
@@ -47,12 +52,24 @@
 @Controller
 public class MainController {
 
-	@Inject private DateManagerService dateManagerService;
+    @Inject private DateManagerService dateManagerService;
+	
+    @Autowired
+    private SessionManager sessionManager;
+    
+    @Autowired
+    private PreferencesService preferencesService;
+
+    @RequestMapping(value = {"/", "/index"}, method = RequestMethod.GET)
```

Comment:
You can use GetMapping here. Bit simpler.

Reply:
Hi, 
Thank you. I've made the change to GetMapping.